renku/cli/show.py

```diff
@@ -151,29 +168,28 @@ def siblings(client, revision, flat, verbose, paths):
 
 @show.command()
 @click.option("--revision", default="HEAD")
-@click.argument(
-    "paths", type=click.Path(exists=True, dir_okay=False), nargs=-1,
-)
+@click.option("-v", "--verbose", is_flag=True)
```

Comment:
should we add a `-c`/`--columns` flag similar to other commands?

Reply:
I don't think it's required here because the information that we show are limited where all of them fit in the screen.